package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "validate-steuerid",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
@UgurGumushan on npm the version is still 1.0.1, so I don't think changing it to 1.0.3 is correct.
https://www.npmjs.com/package/validate-steuerid

Reply:
After we merge and you run npm publish it should be updated there. (Maybe I am missing something)

Reply:
I'm saying that the version should be 1.0.2 instead, cause your code shows that we skipped this version (went from 1.0.1 to 1.0.3 straight)

Reply:
Ah yes, I needed to avoid conflicts on my local. Can do 1.0.2

Reply:
[Done](https://github.com/kontist/validate-steuerid/pull/3/commits/5390835d942845dc9384debb8353a9891da30d8b)